src/Microsoft.Diagnostics.Runtime/CustomDataTarget.cs

```diff
@@ -36,11 +37,14 @@ public class CustomDataTarget : IDisposable
         /// </summary>
         public string? DefaultSymbolPath { get; set; }
 
+        public TokenCredential? SymbolTokenCredential { get; set; }
+
         /// <summary>
         /// Constructor.
         /// </summary>
         /// <param name="reader">A non-null IDataReader.</param>
-        public CustomDataTarget(IDataReader reader)
+        /// <param name="symbolCredential">The TokenCredential to use for any Azure based symbol servers (set to null if not using one).</param>
+        public CustomDataTarget(IDataReader reader, TokenCredential? symbolCredential)
```

Comment:
I think you forgot to set the SymbolTokenCredential property with the symbolCredential parameter here.  It would be nice if you set this new parameter to null so it doesn't break SOS.

Reply:
I did, thank you.  Fixed here, including setting the default to null:  https://github.com/microsoft/clrmd/pull/1261

I'm racing as fast as possible to get these changes in, so I missed that.  :(